code/Base/Hud/Chat/ChatBox.cs

```diff
@@ -0,0 +1,137 @@
+﻿using Sandbox.UI.Construct;
+using Sports.PartySystem;
+using System;
+
+namespace Sports.UI;
+[UseTemplate]
+public partial class SportsChatBox : Panel
+{
+
+	/// <summary>
+	/// disable chat using ClientVar
```

Comment:
Docs should be well-formatted and include grammar. Punctuation is nice but I'm not going to be that anal about it. If a param, value, etc tag is empty, either fill it or remove it.